go.mod

```diff
@@ -25,6 +25,117 @@ require (
 	golang.org/x/text v0.3.7
 )
 
+require (
+	decred.org/cspp/v2 v2.0.0 // indirect
```

Comment:
run `go mod tidy` to remove the indirects

Reply:
Nope it won’t… it came in due to the go version. 1.17 I checked it out already and it is supposed to happen. Same happened to the gioui when it was updated to the a higher go version.

it is a dependency issue.